### ISSUE TITLE
[Item] Benedict's Chest

### ISSUE THREAD

**Issue by DanielJ79** [dalaranwow/dalaran-wow#5430]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After killing Lieutenant Benedict, a key (http://www.wowhead.com/item=4882/benedicts-key#unlocks)  drops from him needed to open the chest (http://www.wowhead.com/object=3239/benedicts-chest#comments)  which contains a letter that begins the quest (http://wowwiki.wikia.com/wiki/Quest:The_Admiral%27s_Orders). Last summer this quest worked perfectly, but now the letter is missing, i abandoned this quest a lot of times and then start again but the letter was still missing.

**Expected behaviour**: Tell us what should happen instead
When you get the key, you go to the chest open it and there is a letter that begins the quest (http://wowwiki.wikia.com/wiki/Quest:The_Admiral%27s_Orders). 
**Steps to reproduce the problem**:

1. Put the letter in the chest.
2. Make it to respawn.
3. Resolve this, because this quest worked perfectly last year.

**Include proofs for this behaviour**
I have a forsaken character, but i played as orc 2-3 months ago i remember that this issue was still present.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills** http://www.wowhead.com/item=4882/benedicts-key#dropped-by, http://wowwiki.wikia.com/wiki/Benedict%27s_Chest. 

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/. I introduced all links.

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Tested in-game just now and it works like a charm, this was never broken as far as I know. Make sure you meet the criteria and that you haven't done the quest already.

![image](https://user-images.githubusercontent.com/5057942/39406777-69407544-4bbc-11e8-8290-77b1c1d59b6b.png)

Closing
